lib/matplotlib/tests/test_font_manager.py

```diff
@@ -275,11 +276,10 @@ def bad_idea(n):
 
 def test_fontcache_thread_safe():
     pytest.importorskip('threading')
-    import inspect
 
-    proc = subprocess.run(
-        [sys.executable, "-c",
-         inspect.getsource(_test_threading) + '\n_test_threading()']
+    proc = subprocess_run_helper(
+        _test_threading,
+        timeout=10
     )
     if proc.returncode:
```

Comment:
This check can be removed, I think, because `subprocess_run_helper` uses `check=True`, so you'd never get here.